src/app/shared/timesheet.service.spec.ts

```diff
@@ -1,12 +1,56 @@
 import { TestBed } from '@angular/core/testing';
 
+import * as _ from 'Lodash';
+
 import { TimesheetService } from './timesheet.service';
+import { Timesheet } from './timesheet.model';
+
+describe('TimesheetService - ', () => {
+
+  let service: TimesheetService;
+  const timesheet = {
+    consultant: { email: 'tester@test.com', name: 'tester', },
+    mission: { client: 'Test.com', title: 'Testin' },
+    workingDays: {
+      '0.1900': [0.5, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0],
+    },
+    invoice: null
+  };
+  const token = (mode: string) => btoa(unescape(encodeURIComponent(JSON.stringify({ mode, timesheet }))));
 
-describe('TimesheetService', () => {
   beforeEach(() => TestBed.configureTestingModule({}));
 
   it('should be created', () => {
-    const service: TimesheetService = TestBed.get(TimesheetService);
+    service = TestBed.get(TimesheetService);
     expect(service).toBeTruthy();
   });
+
```

Comment:
Unit tests are not correct.
- They should describe the API's public methods
- They should test what the service actually does